extensions/amp-subscriptions/0.1/local-subscription-platform.js

```diff
@@ -160,7 +160,10 @@ export class LocalSubscriptionPlatform {
    * @param {!./amp-subscriptions.RenderState} renderState
    */
   activate(renderState) {
-    this.renderer_.render(renderState);
+    this.urlBuilder_.setAuthResponse(renderState.entitlement);
+    this.actions_.build().then(() => {
+      this.renderer_.render(renderState);
```

Comment:
https://github.com/ampproject/amphtml/blob/master/extensions/amp-subscriptions/0.1/amp-subscriptions.css#L64-L66

does this using attributes. Any reason to switch to use classes instead?

Reply:
You mean for `i-amphtml-subs-display`? It could be a class yeah - I don't think it'd make a lot of difference, but up to you. I can't think of any bad consequences at this point. We have been generally using attributes for this kind of thing in fear that someone can find a way to just wipe the whole class. E.g. `element.className = ''`. But I don't know how real that is in this case.

Reply:
Done